cfgov/cdntools/backends.py

```diff
@@ -72,13 +80,7 @@ def post(self, url, action):
 
     def post_tags(self, tags, action):
         """Request a purge by cache_tags."""
-        _url = os.getenv("AKAMAI_FAST_PURGE_URL")
-        if not _url:
-            logger.info(
-                "Can't purge cache. No value set for 'AKAMAI_FAST_PURGE_URL'"
-            )
-            return
-        url = _url.replace("url", "tag")
+        url = self.fast_purge_url.replace("url", "tag")
```

Comment:
I realize this is just carrying over the existing logic, but this risks replacing the characters "url" should they ever exist elsewhere in the fast purge URL (although I realize they don't right now). What about something like
```suggestion
        if not (match := re.match(r"(.*)/url/([^/]+)?$", self.fast_purge_url)):
            raise ValueError("Fast purge URL doesn't match expected format")
        url = f"{match[1]}/tag/{match[2]}"
```
One could theoretically put this in some kind of `fast_purge_tag_url` property.

Reply:
I almost think I'd prefer to take an independent cache tag URL configuration value, so everything is explicit rather than based on the assumption that s/url/tag works to purge cache tags (which, it does, for now). How would you feel about punting that change for after the cutover when we would only have to change those values in one place?

Reply:
Sounds reasonable!